src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -517,13 +520,17 @@ async fn healthcheck(config: CloudwatchLogsSinkConfig, resolver: Resolver) -> cr
 }
 
 fn create_client(
-    region: Region,
-    assume_role: Option<String>,
+    config: &CloudwatchLogsSinkConfig,
     resolver: Resolver,
 ) -> crate::Result<CloudWatchLogsClient> {
```

Comment:
Since the primary parameter to this is `&CloudwatchLogsSinkConfig` would it make sense for this to be a method on that struct instead (and replace `config` with `&self`)? Same question for many of the other `create_client` functions below.

Reply:
Great suggestion! Moved.